circuit/src/gadgets/range.rs

```diff
@@ -131,14 +131,11 @@ impl<F: FieldExt, const K: usize, const N: usize> MockChip<F> for LookupRangeChe
 			// the difference of the running sums:
 			//    z_i = 2^{K}⋅z_{i + 1} + a_i
 			// => a_i = z_i - 2^{K}⋅z_{i + 1}
-			let running_sum_lookup = {
-				let running_sum_word = {
-					let z_next = meta.query_advice(running_sum, Rotation::next());
-					z_cur.clone() - z_next * F::from(1 << K)
-				};
-
-				running_sum_selector.clone() * running_sum_word
+			let running_sum_word = {
+				let z_next = meta.query_advice(running_sum, Rotation::next());
+				z_cur.clone() - z_next * F::from(1 << K)
 			};
```

Comment:
This is still a nested block

Reply:
Done https://github.com/eigen-trust/protocol/pull/169/commits/4cf0c213db9f7092b70767861e6e29ca630aad05